csharp-linq/SystemDynamicLinqCoreLibrary/SystemDynamicLinqCoreLibrary/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Linq.Dynamic.Core;
+using System.Linq.Expressions;
+using SystemDynamicLinqCoreLibrary;
+
+var employee = new Employee();
+
+var employees = employee.GetEmployees();
+
+var itDepartmentEmployees = employees.Where("Department == @0", "IT").ToList();
+
+var itEmployees = employees.Where("Department == \"IT\"").ToList();
+
+var itEmployeesList = employees.Where("c => c.Department = \"IT\"").ToList();
+
+var relevantData = employees.Select("new {Name, Department}").ToDynamicList();
+
+var sortedData = employees.OrderBy("Name").ToList();
+
+var multipleSortedData = employees.OrderBy("Department desc, Name").ToList();
+
+
```

Comment:
All the different examples should be in methods and separated form the Program class. This will help test them in the test project.

Reply:
I have moved all this logic to a separate class.

Reply:
Excellent!